src/components/FlowRunPopOver.vue

```diff
@@ -84,4 +37,9 @@
   w-full
   h-full
 }
+
+.flow-run-pop-over__content { @apply
+  p-2
+  pointer-events-none
```

Comment:
Won't this prevent interacting with the contents of the popover? So I wouldn't be able to hover a point and then go to the run associated with it?

Reply:
hmm I feel like you are correct, I wanted to add some spacing between the trigger and the content but not have it as part of the interaction. But I think you're right this will cascade to the content itself. I'll double check